client/src/app/pages/controls/tags/tags.tsx

```diff
@@ -157,281 +152,283 @@ export const Tags: React.FC = () => {
+    getTagCategoryFallbackColor(tagCategoryToUpdate);
   return (
     <>
       <ConditionalRender
         when={isFetching && !(tagCategories || fetchError)}
         then={<AppPlaceholder />}
       >
-        <AppTableWithControls
-          paginationProps={paginationProps}
-          paginationIdPrefix="tags"
-          count={tagCategories ? tagCategories.length : 0}
-          sortBy={sortBy}
-          onSort={onSort}
-          onCollapse={collapseRow}
-          cells={columns}
-          rows={rows}
-          isLoading={isFetching}
-          loadingVariant="skeleton"
-          fetchError={fetchError}
-          toolbarClearAllFilters={handleOnClearAllFilters}
-          toolbarToggle={
-            <FilterToolbar
-              filterCategories={filterCategories}
-              filterValues={filterValues}
-              setFilterValues={setFilterValues}
-            />
-          }
-          toolbarActions={
-            <ToolbarGroup variant="button-group">
-              <RBAC
-                allowedPermissions={controlsWriteScopes}
-                rbacType={RBAC_TYPE.Scope}
-              >
-                <ToolbarItem>
-                  <Button
-                    type="button"
-                    id="create-tag"
-                    aria-label="Create tag"
-                    variant={ButtonVariant.primary}
-                    onClick={() => setTagModalState("create")}
-                  >
-                    {t("actions.createTag")}
-                  </Button>
-                </ToolbarItem>
-                <ToolbarItem>
-                  <Button
-                    type="button"
-                    id="create-tag-category"
-                    aria-label="Create tag category"
-                    variant={ButtonVariant.secondary}
-                    onClick={() => setTagCategoryModalState("create")}
+        <div
+          style={{
+            backgroundColor: "var(--pf-v5-global--BackgroundColor--100)",
+          }}
+        >
+          <Toolbar {...toolbarProps}>
+            <ToolbarContent>
+              <FilterToolbar {...filterToolbarProps} />
+              <ToolbarGroup variant="button-group">
+                <RBAC
+                  allowedPermissions={controlsWriteScopes}
+                  rbacType={RBAC_TYPE.Scope}
+                >
+                  <ToolbarItem>
+                    <Button
+                      type="button"
+                      id="create-tag"
+                      aria-label="Create tag"
+                      variant={ButtonVariant.primary}
+                      onClick={() => setTagModalState("create")}
+                    >
+                      {t("actions.createTag")}
+                    </Button>
+                  </ToolbarItem>
+                  <ToolbarItem>
+                    <Button
+                      type="button"
+                      id="create-tag-category"
+                      aria-label="Create tag category"
```

Comment:
the label is not translated

Reply:
To be snarky and pedantic -- it's not externalized